sway/desktop/render.c

```diff
@@ -1757,11 +1842,16 @@ void output_render(struct sway_output *output, struct timespec *when,
 		fullscreen_con = workspace->current.fullscreen;
 	}
 
-	int width, height;
-	wlr_output_transformed_resolution(wlr_output, &width, &height);
 
-	fx_renderer_begin(renderer, width, height);
+	struct wlr_box monitor_box = get_monitor_box(wlr_output);
+	wlr_box_transform(&monitor_box, &monitor_box,
+			wlr_output_transform_invert(wlr_output->transform),
+			monitor_box.width, monitor_box.height);
+
+	fx_renderer_begin(renderer, monitor_box.width, monitor_box.height);
```

Comment:
This would be cleaner if there was a way to just get the width and height of the transform invert output instead of the entire box. Haven't found a way based on a quick scroll through wlroots